src/components/PaginatedTable/PaginatedTable.css

```diff
@@ -39,4 +39,17 @@
 
 .paginated-table-pagination {
   margin: 0;
+  padding-left: var(--paginated-table-pagination-padding-left, 0);
```

Comment:
@branquinhoaa let's change this to refer to the `padding` CSS shorthand instead of just `padding-left`. This will give the user a bit more flexibility, if needed.